lib/rust/ensogl/core/src/display/navigation/navigator.rs

```diff
@@ -29,25 +41,34 @@ pub struct NavigatorModel {
     /// Indicates whether events handled the navigator should be stopped from propagating further
     /// after being handled by the Navigator.
     disable_events:  Rc<Cell<bool>>,
+    max_zoom_limit:  Rc<Cell<Option<f32>>>,
 }
 
 impl NavigatorModel {
     pub fn new(scene: &Scene, camera: &Camera2d) -> Self {
         let zoom_speed = Rc::new(Cell::new(Switch::On(10.0 / 1000.0)));
         let pan_speed = Rc::new(Cell::new(Switch::On(1.0)));
-        let min_zoom = 10.0;
-        let max_zoom = 10000.0;
         let disable_events = Rc::new(Cell::new(true));
+        let max_zoom_limit = default();
         let (simulator, resize_callback, _events) = Self::start_navigator_events(
             scene,
             camera,
-            min_zoom,
-            max_zoom,
+            MIN_ZOOM,
+            MAX_ZOOM,
```

Comment:
We can just remove those parameters and use constants in place of their usages directly. Then probably the confusion with line `let max_zoom = max_zoom_limit.get().unwrap_or(max_zoom);` should be mended (especially after renaming MAX_ZOOM to DEFAULT_MAX_ZOOM).

Reply:
Yeah, I missed that initially, and only wake with the exact same thought today :)

Reply:
Fixed